### PR TITLE
Add `FadeInImage.assetNetwork` example

### PR DESCRIPTION
**Pull request by swashcap**
* **Problem:** It's difficult to know how to use a bundled asset as a `FadeInImage` placeholder (see flutter/flutter#18333
* **Solution:** Add an example!
* **Testing:** ???

Closes flutter/flutter#18333.

### CODE REVIEW

**Comment by swashcap** [comment]
Is there an easy way to make a GIF like the memory example? Or, should I just create a starter Flutter project and add the “complete example” code?

**Comment by Hixie** [comment]
cc @Sfshaza

**Comment by Sfshaza** [comment]
@swashcap, thanks for filling a hole in the cookbook!

Travis failed on the commit with this error:

  error • The argument type 'String' can't be assigned to the parameter type 'Uint8List' • example/fading_in_images_md_2.dart:21:13

The complete example looks fine to my naked eye, but something is unhappy.

@brianegan, please review and weigh in.

**Comment by Sfshaza** [comment]
I just checked the fade-in example and, @brianegan, it's HUGE. Can it be reduced in size?

@swashcap, I don't know how Brian did it, but we have [these instructions on the Flutter wiki](https://docs.google.com/spreadsheets/d/1_mXxBKc9G9zCN1KNmUDSLrVoMFdFNnPGTtNofva775M/edit#gid=0).

File: cookbook/images/fading-in-images.md [hunk 1]
```diff
@@ -68,3 +68,61 @@ class MyApp extends StatelessWidget {
...
+    final title = 'Fade in images';
+
+    return new MaterialApp(
+      title: title,
+      home: new Scaffold(
+        appBar: new AppBar(
+          title: new Text(title),
+        ),
+        body: Center(
+          child: new FadeInImage.memoryNetwork(
```
Comment (brianegan) [inline cookbook/images/fading-in-images.md]: Hey there, thanks for the contribution!

It looks like this is the line that's causing the build to fail. I think if ya change this line to `FadeInImage.assetNetwork(...`, you should be all good!

Comment (swashcap) [reply cookbook/images/fading-in-images.md]: Oh, whoops! Yep, I can do that.

**Comment by brianegan** [comment]
@Sfshaza Yep! Looks like that one is crazy huge. I was told a redesign was coming soon and we'll have a better spot for content like this, but in the meantime I'll ensure we don't have any monsters like this one.

File: cookbook/images/fading-in-images.md (hunk 1 above)
Comment (swashcap) [reply cookbook/images/fading-in-images.md]: Fixed in 38a1491a5ae846944c68daf020b33815a8361c2f.

**Comment by swashcap** [comment]
> It would be great to get rid of the `new` keyword from the complete example!

Removed in cbf18ea32e36a695624f4012de5e38e565825061.

I also added a demo GIF in c351351433b22f36906daffb66721db914307771! :tada:

**Comment by brianegan** [comment]
Thanks for making those updates @swashcap. Looks good on my end!

**Comment by Sfshaza** [comment]
I take that as an LGTM!

**Comment by brianegan** [comment]
Thanks @Sfshaza :)

**Pull request merged**